app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/home/livecamera/ClientLiveCameraFragment.kt

```diff
@@ -98,7 +100,14 @@ class ClientLiveCameraFragment : BaseDaggerFragment() {
             RtcConnectionState.Completed -> streamProgressBar.visibility = View.GONE
             RtcConnectionState.Checking -> streamProgressBar.visibility =
                 View.VISIBLE
+            RtcConnectionState.Error -> handleBabyDeviceSdpError()
             else -> Unit
         }
     }
+
+    private fun handleBabyDeviceSdpError() {
+        showSnackbarMessage(R.string.stream_error)
+        requireActivity()
+            .findNavController(R.id.clientDashboardNavigationHostFragment).navigateUp()
```

Comment:
Would regular `requireActivity().onBackPressed()` not be enough in this use case?